mlir/lib/Dialect/LLVMIR/IR/GENXOps.cpp

```diff
@@ -15,6 +15,15 @@
 
 using namespace mlir;
 
+//===----------------------------------------------------------------------===//
+// genx.matrix.dpas
+//===----------------------------------------------------------------------===//
+
+LogicalResult GENX::MatrixDPASOp::verify() {
+  // TODO
```

Comment:
I would like to first ensure `genx.matrix.dpas` representation is good before working on the verification code.